app/src/main/res/layout/activity_search.xml

```diff
@@ -1,131 +1,137 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/app_bg">
 
-    <FrameLayout
-        android:id="@+id/search_fragment_layout"
+    <RelativeLayout
```

Comment:
The changes look good to me. I think we should convert this layout to use a ConstraintLayout instead of RelativeLayout. Would you like to take that up? I am happy to merge the changes anyways.

Reply:
Yeah, converting the layout to constraint layout will be good. I'll take it up and push the changes asap.